addRemove.test.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { addByOrder, getStorage, remove } from './src/addRemove';
+
+document.body.innerHTML = '<div>'
++ '  <ul id="list"></ul>'
++ '</div>';
+
+describe.only('Add new Task', () => {
```

Comment:
- [ ] (optional) In the test file, there are multiple occurrences of `describe`.only. The `.only` modifier makes Jest run only the tests inside that particular describe block and skips all other test suites. While this can be useful during debugging, it's not recommended to leave it in the final version of the test file, as it might lead to unintentionally skipping other tests during future test runs.
- [ ] To fix this, simply remove the .only modifiers from both describe blocks, so they look like this:

```
describe('Add new Task', () => {
  // Test cases for adding tasks
});

describe('Remove Task', () => {
  // Test cases for removing tasks
});
```